### DISCUSSION TITLE
How to add time-dependent displacement or velocity on material points?

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#238] [Q&A]
I would like to know how to apply time-dependent velocity v(t)=sin(wt) or displacement u(t)=sin(wt) on material points. What's the command in the ".yaml" file?

**Comment by djlittl** [comment]
Please have a look at the example problems that are distributed with Peridigm, in the examples directory.  They are intended to provide guidance on basic operations like this.  Both tensile_test and twist_and_pull have time-dependence boundary conditions.
